iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateVoltageLevelTopology.java

```diff
@@ -36,48 +36,67 @@ public class CreateVoltageLevelTopology extends AbstractNetworkModification {
 
     private final String voltageLevelId;
 
+    private final int lowBusbarIndex;
     private final int busbarCount;
+    private final int lowSectionIndex;
     private final int sectionCount;
 
     private final String busbarSectionPrefixId;
     private final String switchPrefixId;
 
     private final List<SwitchKind> switchKinds;
 
-    CreateVoltageLevelTopology(String voltageLevelId, Integer busbarCount, Integer sectionCount,
+    CreateVoltageLevelTopology(String voltageLevelId, int lowBusbarIndex, Integer busbarCount,
+                               int lowSectionIndex, Integer sectionCount,
                                String busbarSectionPrefixId, String switchPrefixId, List<SwitchKind> switchKinds) {
         this.voltageLevelId = Objects.requireNonNull(voltageLevelId, "Undefined voltage level ID");
-        this.busbarCount = checkCount(busbarCount, "busBar");
-        this.sectionCount = checkCount(sectionCount, "section");
+        this.lowBusbarIndex = checkCount(lowBusbarIndex, "low busbar index");
+        this.busbarCount = checkCount(busbarCount, "busBar count");
```

Comment:
A bit strange to allow 0 busbarCounte or sectionCount (expected switch kinds count: -1)

Reply:
Fixed